.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,35 @@
+name: Benchmark GTSFM on select datasets
+on: [pull_request]
+jobs:
+  door-dataset-sift:
+    name: Run GTSFM on the door dataset using SIFT front-end
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    env:
+      PYTHON_VERSION: 3.8
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniconda-version: "latest"
+          activate-environment: gtsfm-v1
+          environment-file: environment_linux.yml
+          python-version: 3.8
+      - name: Environment setup
+        run: |
+          bash .github/scripts/setup.sh
+          pip install hydra-core==1.1.0
```

Comment:
Thanks for the PR, @ayushbaid. Why do we need the separate pip install here? it should be in the conda env, right?

Reply:
Removed it. For some reason it was not able to find hydra and this line made it work. But now we have it working without any issues.